test/functional/p2p_sendheaders.py

```diff
@@ -406,6 +408,8 @@ def test_nonnull_locators(self, test_node, inv_node):
             test_node.wait_for_block(new_block_hashes[-1])
 
             for i in range(3):
+                self.log.debug("Part 3: i == {}".format(i))
+
                 # Mine another block, still should get only an inv
```

Comment:
What I meant was to transfer those comments from `# ...` to `self.log.debug(...`